packages/picasso.js/src/core/scene-graph/display-objects/path.js

```diff
@@ -42,6 +68,25 @@ export default class Path extends DisplayObject {
       arcGen.cornerRadius(v.desc.slice.cornerRadius);
       const d = arcGen(v.arcDatum);
       this.attrs.d = d;
+    } else if (v.points) {
```

Comment:
move the generating path logic from line nodes to path display objects. See the comment in 'line.js'